docs/_articles/en/lwc/testing.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Test Lighting Web Components
+lang: en
+---
+
+Install [`sfdx-lwc-jest`](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.unit_testing_using_jest_installation) so that you have the Jest test runner for Lightning Web Components ready.
+
+## Run or Debug Jest Tests from Within a File
+
+You can run or debug Jest unit tests in contextual actions within your file.
+
+To run Jest tests, in your `.test.js` file, click **Run Test** above the definition of a Jest test case.
+Results from your test run display in the Output panel.
```

Comment:
Currently, the output of running a test case are showing in output channel. VS Code doesn’t intend to support [colors in output channel](https://github.com/microsoft/vscode/issues/571#issuecomment-538881601). Although it’s possible to colorize output channel by language extension, but it might [conflict with other extensions](https://github.com/Microsoft/vscode-arduino/issues/130). Ideally I would like it to run them in integrated terminal (`vscode.window.createTerminal`), so that it has colorized output and is also more flexible. We don’t have access to the terminal process and events it emitted (exit, data), which is required information of CommandExecution for task and progress views. But we might be able to find out when an execution finished from the test result JSON.

Reply:
^ update on the above, I can use Tasks APIs to execute the test tasks in the integrated terminal. https://code.visualstudio.com/api/references/vscode-api#tasks